it/src/main/java/org/corfudb/universe/node/server/CorfuServerParams.java

```diff
@@ -79,7 +79,7 @@
     private final String dockerImage = DOCKER_IMAGE_NAME;
 
     @Default
-    private final int logSizeQuotaPercentage = 100;
+    private final double logSizeQuotaPercentage = 100;
```

Comment:
PR comment says universe framework does not support double but here the parameter is being changed from an int to double right?

Reply:
yes
I just updated the pr comment.
We need doubles

Reply:
I mean logSizeQuotaPercentage is int but we need double